libsrc/Makefile

```diff
@@ -91,7 +93,7 @@ define INSTALL_recipe
 
 $(if $(prefix),,$(error variable `prefix' must be set))
 $(INSTALL) -d $(DESTDIR)$(datadir)/$(dir)
```

Comment:
I am sorry for not being clear enough. Let's say that we have $prefix set to - say - "/usr/local". This means $datadir will become `/usr/local/share/cc65`. That's fine. Now we supply `DESTDIR=/my/dest/dir` as specified in https://www.gnu.org/prep/standards/html_node/DESTDIR.html
We end up with `install` lines trying to install things in `/my/dest/dir/usr/local/share/cc65`, which I thought was wrong but apparently that's how it should work as this is similar to the example given in the above URL.